### PR TITLE
Added Reproduction Doc - Need Some Feedbacks

### PR DESCRIPTION
**Pull request by gitHubAndyLee2020**
I added a reproductions doc under a new folder called docs/.

Please let me know if additional changes to the repro doc are needed.

I also need the associated research paper info.

### CODE REVIEW

File: docs/experiments-rankvicuna-rerank.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
+# RankLLM: RankVicuna Re-rank Collections
+
+This guide provides instructions to reproduce the RankVicuana Re-rank model described in the following paper.
```
Comment (ronakice) [inline docs/experiments-rankvicuna-rerank.md:3]: RankVicuna.

File: docs/experiments-rankvicuna-rerank.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
+# RankLLM: RankVicuna Re-rank Collections
+
+This guide provides instructions to reproduce the RankVicuana Re-rank model described in the following paper.  
+
+Note, we recommend using >24GB RAM for top-20 ranking, and >30GB RAM for top-100 ranking experiments.  
+Additionally, we recommend using A100 with high-RAM option when working on Colab.  
+The expected runtime for the experiments is around 3-4 mins for top-20 ranking, and 30-45 mins for top-100 ranking.
```
Comment (ronakice) [inline docs/experiments-rankvicuna-rerank.md:7]: Is this for each of dl19/dl29

Comment (gitHubAndyLee2020) [reply docs/experiments-rankvicuna-rerank.md:7]: The time is for each dl19/dl20, I can specify further in the doc.

File: docs/experiments-rankvicuna-rerank.md (hunk 1 above)
Comment (gitHubAndyLee2020) [reply docs/experiments-rankvicuna-rerank.md:3]: Thank you.

**Pull request closed without merging**